src/main/java/seedu/teachbook/logic/Logic.java

```diff
@@ -56,4 +56,9 @@
      */
     void setGuiSettings(GuiSettings guiSettings);
 
+
+    /**
+     * Returns the index of currently selected class
+     */
+    int getCurrentlySelectedClassIndex();
```

Comment:
Hi Shufan (@fans2619), just to check, now that I have got it to work locally with the idea you suggested. I have created a new method inside Logic interface. But now it causes some errors in some stubs for the addCommand. Am I supposed to touch the Logic interface?